hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -90,9 +91,19 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class ExecutionLifecycleTest extends SimpleTestInClusterSupport {
 
+    static class NonSerializableException extends RuntimeException {
+
+        private Optional<Boolean> thing = Optional.of(true);
```

Comment:
This would suffice :)
```suggestion
        private Object nonSerializableField = new Object();
```